src/styles/blocks/catalog.scss

```diff
@@ -18,8 +17,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: ease 300ms;
```

Comment:
specify css property for transition. by default it is `all`, but it is not good for performance